nipype/workflows/dmri/fsl/utils.py

```diff
@@ -125,34 +125,42 @@ def dwi_flirt(name='DWICoregistration', excl_nodiff=False, flirt_param={}):
         fsl.FLIRT(**flirt_param),
         name='CoRegistration',
         iterfield=['in_file', 'in_matrix_file'])
-    thres = pe.MapNode(
-        fsl.Threshold(thresh=0.0),
-        iterfield=['in_file'],
-        name='RemoveNegative')
+    apply_xfms = pe.MapNode(
+        fsl.ApplyXFM(
+            apply_xfm=True,
+            interp='spline',
```

Comment:
At a first sight, using spline interpolation here seems like a good idea but will introduce negative values. Thresholding after this step would be advisable